INSTALL.rst

```diff
@@ -49,12 +56,30 @@ Create a virtual environment and clone the INSPIRE source code using `git`:
 
 .. code-block:: console
 
-    $ mkvirtualenv inspirehep
+    $ mkvirtualenv -p /usr/bin/python2.7 inspirehep
```

Comment:
that will get messed up if you are already in a venv (happened to me at some point when playing with different compilations of python)